packages/editor-ui/src/composables/useDocumentVisibility.ts

```diff
@@ -0,0 +1,52 @@
+import type { Ref } from 'vue';
+import { ref, onMounted, onUnmounted } from 'vue';
+
+type VisibilityHandler = () => void;
+
+interface DocumentVisibilityResult {
```

Comment:
I tend to never use interfaces, out of curiosity where do you draw the line between the two?

Reply:
I am also for types, but was pulling this from another composable we had and forgot to update. Will change so we are more consistent.